op-service/txmgr/txmgr.go

```diff
@@ -790,14 +887,19 @@ func (m *SimpleTxManager) checkLimits(tip, baseFee, bumpedTip, bumpedFee *big.In
 func (m *SimpleTxManager) checkBlobFeeLimits(blobBaseFee, bumpedBlobFee *big.Int) error {
 	// If below threshold, don't apply multiplier limit. Note we use same threshold parameter here
 	// used for non-blob fee limiting.
-	if thr := m.cfg.FeeLimitThreshold; thr != nil && thr.Cmp(bumpedBlobFee) == 1 {
+	m.cfgLock.Lock()
+	feeLimitThreshold := m.cfg.FeeLimitThreshold
+	feeLimitMultiplier := m.cfg.FeeLimitMultiplier
+	m.cfgLock.Unlock()
```

Comment:
This makes sense, to load the values upfront, to have them be the same in the context of this function call. Did you check that all repeated accesses to any config fields within a function are now preloaded in this way?